lib/createDIContainer.js

```diff
@@ -314,6 +324,7 @@ async function createDIContainer(options) {
         isolatedJsonSchemaValidatorSnapshot,
         isolatedSTUnserializationOptions,
         stateRepository,
+        additionalDppOptions,
       );
 
       return unserializeStateTransitionFactory(createIsolatedDpp);
```

Comment:
Why this function is called like this :D There's no such word :D It should be `parse`. I don't know if we should rename it here though

Reply:
Or deserialize, because that's the reverse term to serialization, for that matter

Reply:
Totally agree! Let's do that in a separate PR?